SwedbankPaySDKTests/Utils/FileUtils.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import XCTest
+
+enum FileUtils {
+    private static var emptyBuffer: Void = ()
+    private static func openData(_ content: Data) throws -> UnsafeMutablePointer<FILE> {
+        try content.withUnsafeBytes {
+            let size = $0.count
+            XCTAssert(size > 0)
+            let file = try XCTUnwrap(fmemopen(nil, size, "rb+"))
+            XCTAssertEqual(fwrite($0.baseAddress, size, 1, file), 1)
+            rewind(file)
+            return file
+        }
+    }
+    
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6137886)